python_modules/libraries/dagster-k8s/dagster_k8s/client.py

```diff
@@ -731,6 +731,50 @@ def _has_container_logs(self, container_status):
 
         return False
 
+    def _get_job_status_str(self, job):
+        if not job.status:
+            return "Could not determine job status."
+
+        job_status = (
+            "Job status:"
+            + f"\n - starttime: {job.status.start_time.isoformat()}"
+            + f"\n - active={job.status.active or 'None'}"
+            + f"\n - succeeded={job.status.succeeded or 'None'}"
+            + f"\n - failed={job.status.failed or 'None'}"
+        )
+
+        return job_status
+
+    def get_job_debug_info(
+        self,
+        job_name,
+        namespace,
+        job: Optional[kubernetes.client.V1Job] = None,  # the already fetched job
```

Comment:
nit: could defer adding this until theres a callsite that can use it

Reply:
Changed